crates/matrix-crypto-ffi/src/lib.rs

```diff
@@ -161,4 +161,10 @@ fn parse_user_id(user_id: &str) -> Result<Box<ruma::UserId>, CryptoStoreError> {
         .map_err(|e| CryptoStoreError::InvalidUserId(user_id.to_owned(), e))
 }
 
-include!(concat!(env!("OUT_DIR"), "/olm.uniffi.rs"));
+#[allow(warnings)]
+mod generated {
+    use super::*;
+    include!(concat!(env!("OUT_DIR"), "/olm.uniffi.rs"));
+}
+
+pub use generated::*;
```

Comment:
This is kind of ugly but the only way I could think of to silence the extra warnings in the generated code. Wdyt @poljar @gnunicorn?

Reply:
That's annoying, I presume ignoring based on a pattern would be more annoying or not possible?

Reply:
What kind of pattern do you mean?

Reply:
I mean in some kind of config file, or config switch that takes a pattern which will then exclude files from warnings.

Reply:
I don't think that kind of thing exists. Although we could create a rustc issue about this (since spans get attributed correctly, it seems technically feasible to filter things out).

Meanwhile, are you okay with this workaround (from CI, it looks like it fixed things).

Reply:
> Meanwhile, are you okay with this workaround (from CI, it looks like it fixed things).

Yeah, I'm fine with it.

Reply:
I've brought this up in [#uniffi:mozilla.org](https://matrix.to/#/#uniffi:mozilla.org).